src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -238,6 +240,28 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
             </div>
           </div>
           <div id="info">
+            <div class="step-icons">
+              <ha-icon-button
+                class="minus step-icon ${classMap({
+                  disabled: Array.isArray(this._setTemp) && !this._lastSetMode,
```

Comment:
This is weird, we can better remove the buttons if they are only going to work after interaction with the slider

Reply:
Removing and then adding would either cause the card height to jump or for there to normally be a large gap between the bottom of the slider and the icons